src/Workspaces/Core/Portable/CodeStyle/CodeStyleOption.cs

```diff
@@ -189,6 +189,6 @@ public override bool Equals(object obj)
                Equals(option);
 
         public override int GetHashCode()
-            => Hash.Combine(Value.GetHashCode(), Notification.GetHashCode());
+            => unchecked((Notification.GetHashCode() * (int)0xA5555529) + Value.GetHashCode());
```

Comment:
is HashCode.Combine available for us to use now?

Reply:
The Code Style layer is .NET Standard 1.3.

Reply:
I don't know what that means.  But i assume it's synonymous with "no" :D

Reply:
[That is correct](https://apisof.net/catalog/System.HashCode)

Reply:
What about using `(Notification, Value).GetHashCode()`? Are tuples unavailable here?

Reply:
Tuples are available, but this pull request was a behavior-preserving refactoring and the implementation of `ValueTuple<T1, T2>.GetHashCode()` is not the same as the one in **Hash.cs**.